src/api/emulations/nfs.js

```diff
@@ -1,12 +1,33 @@
 const lib = require('../../native/lib');
 const t = require('../../native/types');
 const nativeH = require('../../native/helpers');
+const consts = require('../../consts');
 
 function isString(arg) {
   return typeof arg === 'string' || (arg.toString ? arg.toString() === '[object String]' : false);
 }
 
 /**
+* Create a new file
+* @returns {File}
+**/
+function newFile() {
```

Comment:
I originally had it as a method of NFS, however, the linter was complaining that the method didn't use `this`